department-of-veterans-affairs/abd-vro
# MCP-2426 Default PDF Template

## What was the problem?

If a user put in a code that we do not have a template for, the PDF Generator will fail

Associated tickets or Slack threads:

- [MCP-2426](https://amida.atlassian.net/browse/MCP-2426)

## How does this fix it?[^1]

If the code template does not exist, it will put all the request data inside the default template and render that PDF instead

## How to test this PR
- Generate a PDF using a random `diagnosticCode` or any text.
- Generation should be successful and when you open the PDF, you should see something similar to the screenshot below which will contain all the info from the request
<img width="1093" alt="image" src="https://user-images.githubusercontent.com/13013464/223906435-2b3929c3-c013-4d24-9d2f-378f0ee15520.png">



[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
